README.md

```diff
@@ -953,6 +953,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Sanskriti Gupta](https://github.com/sanskritigupta5)
   - [Sanyam Gupta](https://github.com/sanyam19)
   - [Sarthak Dhonchak](https://github.com/SarthakDhonchak)
+  - [Sarthak Majorwar].(https://github.com/Majorwar)
```

Comment:
There's an extra dot here that should be removed.
```suggestion
  - [Sarthak Majorwar](https://github.com/Majorwar)
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable